test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Dynamo.Models;
+using Dynamo.Selection;
+using Dynamo.Tests;
+using Dynamo.ViewModels;
+using DynamoShapeManager;
+using NUnit.Framework;
+using ProtoCore.Mirror;
+using TestServices;
+
+namespace Dynamo
+{
+    public class DynamoModelTestBase : UnitTestBase
+    {
```

Comment:
We really needed this.  Many of our tests unnecessarily also create a `DynamoViewModel`.